__mocks__/react-dom.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const reactDOM = jest.genMockFromModule('fs'); // eslint-disable-line no-undef
+
+// Overwrite this in the test to mock it.
+reactDOM.__findDOMNode = () => {};
+
+reactDOM.findDOMNode = function(...args) {
+  return reactDOM.__findDOMNode(...args);
+};
+
+module.exports = reactDOM;
```

Comment:
I think you missed the first part of my comment :/ I really think this mock is overkill and the same thing can be done in a simpler way...